server/views/pages/component.njk

```diff
@@ -80,16 +80,74 @@
         </table>
       </div>
     {% endif %}
+
+    {% set upgradeNeeded = false %}
```

Comment:
You could do this in the route rather then the template then just pass `upgradeNeeded` through in the `res.render` ?